exercises/pascals-triangle/test/test_pascals_triangle.c

```diff
@@ -0,0 +1,82 @@
+#include <stdbool.h>
+#include "../src/pascals_triangle.h"
+#include "vendor/unity.h"
+
+static bool check(size_t count, size_t expected[][count], size_t ** result)
+{
+   size_t i, j;
+   for (i = 0; i < count; i++) {
+      for (j = 0; j < count; j++) {
+         if (expected[i][j] != result[i][j]) {
+            return 0;
+         }
+      }
+   }
+   return 1;
+}
+
+void test_no_rows(void)
+{
+   size_t expected[1][1] = { {0} };
+   TEST_ASSERT_TRUE(check(1, expected, pascals_triangle("0")));
+}
+
+void test_single_row(void)
+{
+   size_t expected[1][1] = {
+      {1}
+   };
+   TEST_ASSERT_TRUE(check(1, expected, pascals_triangle("1")));
+}
+
+void test_two_rows(void)
+{
+   size_t expected[2][2] = {
+      {1, 0},
+      {1, 1}
+   };
+   TEST_ASSERT_TRUE(check(2, expected, pascals_triangle("2")));
+}
+
+void test_three_rows(void)
+{
+   size_t expected[3][3] = {
+      {1, 0, 0},
+      {1, 1, 0},
+      {1, 2, 1}
+   };
+   TEST_ASSERT_TRUE(check(3, expected, pascals_triangle("3")));
+}
+
+void test_four_rows(void)
+{
+   size_t expected[4][4] = {
+      {1, 0, 0, 0},
+      {1, 1, 0, 0},
+      {1, 2, 1, 0},
+      {1, 3, 3, 1}
+   };
+   TEST_ASSERT_TRUE(check(4, expected, pascals_triangle("4")));
+}
+
+void test_negative_rows(void)
+{
+   TEST_ASSERT_TRUE((pascals_triangle("-1") == (size_t **) - 1));
+}
+
+void test_null_row_is_not_allowed(void)
+{
+   TEST_ASSERT_TRUE((pascals_triangle("") == (size_t **) - 1));
+}
+
+int main(void)
+{
+   UnityBegin("test_pascals_triangle.c");
```

Comment:
All the other tests seem to write this as `UnityBegin("test/test_pascals_triangle.c");`

Admittedly, I don't know which way is correct though.